src/libs/actions/Report.ts

```diff
@@ -175,10 +176,27 @@ const typingWatchTimers: Record<string, NodeJS.Timeout> = {};
 
 let reportIDDeeplinkedFromOldDot: string | undefined;
 Linking.getInitialURL().then((url) => {
-    const params = new URLSearchParams(url ?? '');
-    const exitToRoute = params.get('exitTo') ?? '';
-    const {reportID} = ReportUtils.parseReportRouteParams(exitToRoute);
-    reportIDDeeplinkedFromOldDot = reportID;
+    const isWeb = ([CONST.PLATFORM.WEB] as unknown as string).includes(getPlatform());
```

Comment:
this code is not in compliance with our standards for cross-platform code. @rezkiy37 @abdulrahuman5196 @cristipaval please review [App Philosophy](https://github.com/Expensify/App?tab=readme-ov-file#Philosophy) with particular emphasis on item 5. Then please open a follow-up PR to implement this in a way that's compliant with our code standards. Thanks!

Reply:
Okay. I will prepare an additional PR. Ty!

Reply:
I am preparing [a PR](https://github.com/Expensify/App/pull/37754) to fix it.